apps/website/src/pages/join-us/[ashbyId].tsx

```
@@ -13,46 +11,48 @@ import {
-      {data?.job && (
+      {job && (
         <>
           <Head>
-            <title>{`${data?.job?.title} | BlueDot Impact`}</title>
-            <meta name="description" content={data?.job?.subtitle} />
+            <title>{`${job?.title} | BlueDot Impact`}</title>
+            <meta name="description" content={job.location} />
             <script
               type="application/ld+json"
               // eslint-disable-next-line react/no-danger
               dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)